RevenueCatUI/CustomerCenter/Data/CustomerCenterActionViewModifier.swift

```diff
@@ -47,6 +47,8 @@ struct CustomerCenterActionViewModifier: ViewModifier {
     @State private var refundRequestStarted: UniqueWrapper<String>?
     @State private var refundRequestCompleted: UniqueWrapper<(String, RefundRequestStatus)>?
     @State private var feedbackSurveyCompleted: UniqueWrapper<String>?
+    // Management option selected event
+    @State private var managementOptionSelected: CustomerCenterManagementOptionWrapper?
```

Comment:
Have you tried this?
```suggestion
    @State private var managementOptionSelected: UniqueWrapper<CustomerCenterActionable>?
```

This way we wouldn't need `CustomerCenterManagementOptionWrapper`